lib/modules/datasource/deno/index.spec.ts

```diff
@@ -0,0 +1,200 @@
+import * as httpMock from '../../../../test/http-mock';
+import { DenoDatasource } from '.';
+
+describe('modules/datasource/deno/index', () => {
+  const deno = new DenoDatasource();
+
+  describe('getReleases', () => {
+    it('returns releases of standard library', async () => {
+      httpMock
+        .scope('https://apiland.deno.dev')
+        .get('/v2/modules/std')
+        .reply(200, {
+          versions: ['0.163.0', '0.162.0'],
+          tags: [{ value: 'top_5_percent', kind: 'popularity' }],
+        })
+        .get('/v2/modules/std/0.163.0')
+        .reply(200, {
+          version: '0.163.0',
+          upload_options: {
+            repository: 'denoland/deno_std',
+            ref: '0.163.0',
+            type: 'github',
+          },
+          uploaded_at: '2022-11-08T21:10:21.592Z',
+        })
+        .get('/v2/modules/std/0.162.0')
+        .reply(200, {
+          version: '0.162.0',
+          upload_options: {
+            repository: 'denoland/deno_std',
+            ref: '0.162.0',
+            type: 'github',
+          },
+          uploaded_at: '2022-10-20T12:10:21.592Z',
+        });
+
+      const result = await deno.getReleases({
+        packageName: 'https://deno.land/std@0.157.0/http/server.ts',
```

Comment:
🤔 why do we need the version inside packageName? that will influence caching badly.

Reply:
I have used the full import to centralise the logic for cutting up the import string.      

But you are right this prevents good caching, probably it is the best to let the manager to split it at `@` and provide the first element as package name ( `https://deno.land/std` ).  We have to check for that anyway as no `@` in the import is an implicit `latest`.